### PR TITLE
makes sure all commands are functioning

### PR DESCRIPTION
**Pull request by JasonMannon**

### CODE REVIEW

File: app/models/incoming_message/edit.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def execute
         @standup.update_attributes(editing: false)
 
         @client.message channel: @message['channel'], text: I18n.t('activerecord.models.incoming_message.answer_saved')
-        @client.message channel: @message['channel'], text: @standup.current_question
+        @client.message channel: @message['channel'], text: @standup.current_question unless @standup.complete?
```
Comment (santiagodoldan) [inline app/models/incoming_message/edit.rb]: were you able to reach this line with the standup completed?

File: app/models/user.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ class User < ActiveRecord::Base
   validates :slack_id, uniqueness: true
 
   scope :sort, -> { order('sort_order ASC') }
+  scope :admin, -> { find_by(admin_user: true) }
```
Comment (santiagodoldan) [inline app/models/user.rb]: nice :fuckyeah:

File: app/views/standups/index.html.haml [hunk 3]
```diff
@@ -14,9 +14,9 @@
           .panel-heading
             .text-center
               %b
-              = User.find_by(user_id: standup.user_id).full_name
+              = standup.user.full_name
```
Comment (santiagodoldan) [inline app/views/standups/index.html.haml]: much better :)

File: app/models/incoming_message/edit.rb (hunk 1 above)
Comment (JasonMannon) [inline app/models/incoming_message/edit.rb]: When a user edit's a question after they already finished standup, it would run current_question and break.

Comment (santiagodoldan) [inline app/models/incoming_message/edit.rb]: nevermind

**Pull request merged**